config.template.yaml

```diff
@@ -0,0 +1,58 @@
+HelmRepositoryURL: https://s3-us-west-2.amazonaws.com/dispatch-charts
```

Comment:
There is a LOT more going into config.yaml now.  I was really intending to keep it as small as possible.  Also, I don't see a lot of these new values actually being used.  There certainly aren't any changes to install.go.

Reply:
I keep my opinion on this, indeed I introduce more values into the config.yaml, but I think most of them should be configurable, although we use the default values all the time, like the namespace, release and etc of each services.

We should have a default config.yaml, and I believe with that you may think this is more clean, small and organized. (the default yaml might be large)

Reply:
We just spoke in person, but I think it's good to expose all of the configurable options in one place.  This should be the "default" config so the user only has to override the options they want.  Also, this should be built into the dispatch CLI, so that users don't need download this template.